src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -87,6 +87,8 @@ internal class KnownRoutes
 
         public const string BulkDelete = "$bulk-delete";
         public const string BulkDeleteResourceType = ResourceType + "/" + BulkDelete;
+        public const string BulkDeleteSoftDeleted = "$bulk-delete-soft-deleted";
```

Comment:
Also - do we need a new operation definition for `$bulk-delete-soft-deleted` or can it be part of the same one? Unsure here tbh. 😄

Reply:
It should have its own, I forgot. All different urls should have their own.

Reply:
Added